src/containers/ProfilePage/index.jsx

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+import {
+  Container,
+  Row,
+  Accordion,
+  useAccordionToggle,
+  AccordionContext,
+} from "react-bootstrap";
+import { useForm } from "react-hook-form";
+import UserDetailForm from "../../components/profile/UserDetailForm";
+import UserEmailForm from "../../components/profile/UserEmailForm";
+import UserPasswordForm from "../../components/profile/UserPasswordForm";
+import defaultProfileImage from "../../images/defaultProfileImage.png";
+import "./index.scss";
+
+const SECTINOS = [
+  {
+    label: "User details",
+    componentName: UserDetailForm,
+  },
+  {
+    label: "Email address",
+    componentName: UserEmailForm,
+  },
+  {
+    label: "Change password",
+    componentName: UserPasswordForm,
+  },
+];
+
+function getArrowClass(isCurrentElementSelected) {
+  return isCurrentElementSelected ? "arrow down" : "arrow right";
+}
+
+export default function ProfilePage({ submit }) {
+  const { register, handleSubmit, errors } = useForm();
+
+  const onSave = (data) => {
+    // TODO: save data in the database
+    submit(data);
+  };
+
+  function ToggleButton({ children, eventKey, callback }) {
```

Comment:
don't put components within components. put under imports (in the same file, no need for new file)